src/applications/vaos/tests/e2e/fixtures/MockAppointmentResponse.js

```diff
@@ -42,6 +43,7 @@ export default class MockAppointmentResponse {
     patientHasMobileGfe = false,
     serviceType = 'primaryCare',
     status = 'booked',
+    physicalLocation = null,
```

Comment:
It's not a big deal but I would remove physicalLocation from the contructor and just use the setter method to set this property. Reason, the properties in the constructor contain the bare minimum needed to create a valid appointment. This constructor could stand to be refactored more.

Reply:
Done